src/aks-preview/HISTORY.rst

```diff
@@ -22,6 +22,10 @@ Pending
 * Add `--enable-addon-autoscaling` to the `az aks create` command.
 * Add `--ip-families` to the `az aks update` command.
 
+0.5.175
```

Comment:
We've adopted a new naming pattern, please update this to something like `1.0.0b3` and update the `VERSION` in `setup.py` too.
https://github.com/Azure/azure-cli-extensions/pull/7151#discussion_r1446925040